Source/UltraleapTrackingCore/Private/LeapDeviceWrapper.cpp

```diff
@@ -148,12 +148,12 @@ LEAP_TRACKING_EVENT* FLeapDeviceWrapper::GetInterpolatedFrameAtTime(int64 TimeSt
 		if (Result == eLeapRS_RoutineIsNotSeer)
 		{
 			UE_LOG(UltraleapTrackingLog, Log, TEXT("LeapGetFrameSizeEx failed in FLeapDeviceWrapper::GetInterpolatedFrameAtTime: TimeStamp %lld was in the future"), TimeStamp);
-			return nullptr;
+			return GetFrame();
```

Comment:
This looks like something other than hinting? Was it a bug fix?

Reply:
Yea, this will possibly solve an issue when tracking drops when interpolation does not work